### DISCUSSION TITLE
boiler_return_temperature vs boiler_heat_carrier_return_tc0

### DISCUSSION THREAD

**Discussion by HansRemmerswaal** [emsesp/EMS-ESP32#1024] [General]
**Question**

In HA I see the following two sensors having ALWAYS the same value.
sensor.boiler_return_temperature
sensor.boiler_heat_carrier_return_tc0

Are these two sensors for the same or are they actually different sensors?

**Screenshots**
![image](https://user-images.githubusercontent.com/116959990/213988652-acac2ebb-1209-49c5-b95a-9d6cdaaf97b5.png)

**Device information**
[emsesp_info.txt](https://github.com/emsesp/EMS-ESP32/files/10477636/emsesp_info.txt)

**Additional context**

**Comment by MichaelDvP** [comment]
> sensor.boiler_return_temperature

is from telegram E4 offset 7
> sensor.boiler_heat_carrier_return_tc0

is from telegram 48F offset 6

Maybe one sensor published twice by the bus.

**Comment by HansRemmerswaal** [comment]
Looks like... and thanks for your quick reply. Will close it.
